articles/universal-login/index.md

```diff
@@ -0,0 +1,84 @@
+---
+description: Overview of Universal Login with Auth0
+topics:
+  - login
+  - universal-login
+  - password-reset
+  - guardian
+  - mfa
+  - error-pages
+  - hosted-pages
+contentType: index
+toc: true
+useCase: customize-hosted-pages
+---
+# Auth0 Universal Login
+
+Auth0's Universal Login is the most secure way to authenticate users for your applications. Universal Login centers around your Auth0 login page. The login page appearance and behavior is customizable right from the [Dashboard](${manage_url}). The logo and colors of the login widget can be changed, and in more advanced use cases, the code of the page itself can be modified.
+
+In the Universal Login flow, the user will click a login button or link in your application, which will call the `/authorize` endpoint at Auth0. If there is no session detected for the end user, Auth0 will redirect them to the login page, where they will be able to login or signup using the connections you configure, such as databases or social connections. Once the user is authenticated (or if they were already signed in) the login page will redirect them to your application, along with the requisite credentials.
+
+::: note
+If the incoming authentication request includes a `connection` parameter that uses an external identity provider (such as a social provider), the login page will not display. Instead, Auth0 will direct the user to the [identity provider's](/identityproviders) login page.
+:::
+
+For information on the diffences between Universal Login and traditional embedded login within your application, see [our comparison guide](/guides/login/universal-vs-embedded).
+
+## Implementing Universal Login
+
+In addition to configuring Universal Login for your tenant's applications, you will also need to set up a connection(s) and set up your application in Auth0's dashboard. You will also need to configure your application's code to call Auth0's `/authorize` endpoint in order to trigger Universal Login, and then to deal with the response.
+
+For step by step instructions on setting up your application to use Universal Login, check out our [Quickstart guides](/docs/quickstart).
+
+### Simple Customization
+
+![Customization Settings for Login Page](/media/articles/universal-login/settings.png)
+
+In the [Dashboard](${manage_url}), you can see the settings for your login page by navigating to [Universal Login](${manage_url}/#/login_page) and looking at the tab for the Login Page.
+
+The settings available here are:
+
+* Logo
+* Primary Color
+* Background Color
+
+These settings, once changed, will take effect on your login page if you have not enabled customization of the login page code, or if you have enabled customization but are using the predefined templates and have not changed those options in the code.
```

Comment:
These settings, once changed, will take effect on all your universal login pages if you have not enabled customization of the page's code, or if you have enabled customization but are using the predefined templates and have not changed those options in the code.